Content.Server/NodeContainer/Nodes/PipeNode.cs

```diff
@@ -271,16 +259,23 @@ private void UpdateConnectedDirections()
         {
```

Comment:
Can't comment coz github but you can probably just resolve IEntityManager / TransformComponent above and pass it in here to re-use it.

Reply:
yeah I could, I just got lazy with the pipe UpdateConnectedDirections() functions because IMO they should just be purged (#6469).